x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/views/query_detail.tsx

```diff
@@ -0,0 +1,34 @@
+/*
```

Comment:
Noting here while I review:

> Confirm it correctly fwds search query params

On the initial load, there are no query parameters sent on the API request. Is it relying on defaults?